### PR TITLE
Export private key

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Add page to export the private key of an account.

## Changes
- Added a page that allow for exporting the private key of an account after inputting the wallet password.
- Fixed a missed translation / wrong text issue in the account view.

## Checklist
- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/pages/Account/i18n/da.ts [hunk 1]
```diff
@@ -25,7 +25,13 @@ const t: typeof en = {
             connect: 'Forbind',
             disconnect: 'Fjern',
         },
-        exportPrivateKey: 'Eksportér privat nøgle',
+        exportPrivateKey: {
+            title: 'Eksportér privat nøgle',
+            description: 'Indtast venligst din adgangskode for at vise din private nøgle.',
+            copyDescription: 'Tryk på knappen for at kopiere din private nøgle.',
+            show: 'Vis privat nøgle',
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/i18n/da.ts:32]: private eller **privat**?

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    function validateCurrentPasscode(): Validate<string> {
+        return (currentPasscode) => (currentPasscode !== passcode.value ? tPasscode('incorrectPasscode') : undefined);
+    }
+
+    if (showPrivateKey) {
+        return (
+            <div className="export-private-key-page">
+                <div className="export-private-key-page__description">{t('copyDescription')}</div>
+                <div className="export-private-key-page__relative">
+                    <TextArea value={privateKey} />
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx:48]: The fact there are 2 characters on a third line makes me cry.

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import React, { useState } from 'react';
+import { Validate } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { absoluteRoutes } from '@popup/constants/routes';
+
+export default function ExportPrivateKey() {
+    const nav = useNavigate();
+    const { t: tSetup } = useTranslation('setup');
+    const { t: tPasscode } = useTranslation('changePasscode');
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx:20]: This is fine for now, but maybe we need to restructure these translation.

**Review by shjortConcordium — COMMENTED** [review]
LGTM

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountSettings.scss [hunk 4]
```diff
@@ -40,3 +40,40 @@
...
+    background-color: $color-bg;
+
+    &__description {
+        text-align: center;
+        margin-top: rem(30px);
+    }
+
+    &__relative {
+        position: relative;
+    }
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/AccountSettings/AccountSettings.scss:57]: There is already a global `.relative` just use that.

File: packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx (hunk 3 above)
Comment (orhoj) [reply packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx:20]: Yes, I think these shared translations could be done in a better way than this, but let's keep that for later.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**